SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-or-later

Add missing fcntl header.

--- src/kbdrate.c	2021-07-22 22:35:53.671970814 +1000
+++ src/kbdrate.c	2021-07-22 22:36:10.038096797 +1000
@@ -94,6 +94,7 @@
 };
 
 #include <signal.h>
+#include <fcntl.h>
 
 #include "nls.h"
 #include "version.h"
